dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin2/Sample.java

```diff
@@ -0,0 +1,227 @@
+        paraResults = dtResults.concat("/parallelization-results");
+
+        prioDTLists = dtData.concat("/prioritization-dt-lists");
+        seleDTLists = dtData.concat("/selection-dt-lists");
+        paraDTLists = dtData.concat("/parallelization-dt-lists");
+
+        // New
+        newDTSubjectSource = System.getProperty("user.dir");
+
+        newDTSubject = newDTSubjectSource.concat("/target");
+        newDTTools = buildClassPath(newDTSubjectSource.concat("/lib/*"));
+        newDTLibs = buildClassPath(newDTSubject.concat("/dependency/*"));
+        newDTClass = newDTSubject.concat("/classes");
+        newDTTests = newDTSubject.concat("/test-classes");
+
+        // Results
+        newDTResults = newDTSubjectSource.concat("/results");
+    }
+
+    // Compile & Gather The Dependencies Of The Subject
+    private void gatherDependencies(){
+        TestPluginPlugin.mojo().getLog().info("Gathering Dependencies Of The Subject");
+
+        cli.doMain(new String[]{"compile",
+                "test-compile",
+                "-Dmaven.javadoc.skip=true",
+                "-DskipTests",
+                "dependency:copy-dependencies",
+                "-Drat.skip=true",
+                "-Dcobertura.skip" }, newDTSubjectSource, System.out, System.out);
+        cli.doMain(new String[]{"install",
+                "-fn",
+                "-Dmaven.javadoc.skip=true",
+                "-DskipTests",
+                "dependency:copy-dependencies",
+                "-Drat.skip=true",
+                "-Dcobertura.skip" }, newDTSubjectSource, System.out, System.out);
+    }
+
+    // Setup Test Prioritization On The New Version
+    private void setupTestPrioritization(MavenProject project){
+        TestPluginPlugin.mojo().getLog().info("Moving Test Info From The Old Version To The New Version");
+        try {
+            FileUtils.moveFile(new File(dtResults + "/orig-order.txt"), new File(newDTResults + "/orig-order.txt"));
+            FileUtils.moveFile(new File(dtResults + "/ignore-order.txt"), new File(newDTResults + "/ignore-order.txt"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Removing Failed Tests");
+        try {
+            List<String> origOrder = Files.readAllLines(new File(newDTResults + "/orig-order.txt").toPath());
+            List<String> ignoreOrder = Files.readAllLines(new File(newDTResults + "/ignore-order.txt").toPath());
+            origOrder.removeAll(ignoreOrder);
+
+            Files.write(new File(newDTResults + "/orig-order.txt").toPath(), origOrder);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+        TestPluginPlugin.mojo().getLog().info("Removing Missing Tests");
+
+    }
+
+    // Setup Test Selection On The New Version
+    private void setupTestSelection(MavenProject project){
+        String JAVA_HOME = expandEnvVars("${JAVA_HOME}");
```

Comment:
Minor: Seems more appropriate to make this a static final field. Additionally, could use `System.getProperty("java.home")` (unless you have a reason not to).